.github/workflows/rust.yml

```diff
@@ -11,11 +11,11 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-22.04
+    runs-on: buildjet-4vcpu-ubuntu-2204
     steps:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@1.69.0
-      - uses: Swatinem/rust-cache@v2
+      - uses: joroshiba/rust-cache@v2.5.1
```

Comment:
Not a blocker: this should ideally be under the astria org

Reply:
done